src/components/ClientSideLoggingToolMenu/BaseClientSideLoggingToolMenu.tsx

```diff
@@ -32,9 +35,20 @@ type BaseClientSideLoggingToolProps = {
     onEnableLogging?: () => void;
     /** Path used to display location of saved file */
     displayPath?: string;
+    /** Boolean to know if this was opened via test tools modal */
+    isViaTestToolsModal: boolean;
```

Comment:
We shouldn't need `isViaTestToolsModal` prop at all

Reply:
Ah, now I understand what you mean, thanks

I think we have to move the menuItem with 'View console' button from `BaseClientSideLoggingToolMenu` to `TestToolsModal`

We use roughly the same approach in [TroubleshootPage](https://github.com/Expensify/App/blob/badec944783514b871dad89ca1e19b2a8a26ea01/src/pages/settings/Troubleshoot/TroubleshootPage.tsx#L69), where the parent component is the one responsible for rendering 'View  debug console'  button